### PR TITLE
Update the LoadTesting for stable release.

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

**Review by Yao725 — APPROVED** [review]

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/Generated/Models/LoadTestResourcePageList.Serialization.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ internal partial class LoadTestResourcePageList
     {
         internal static LoadTestResourcePageList DeserializeLoadTestResourcePageList(JsonElement element)
```
Comment (prativen) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/Generated/Models/LoadTestResourcePageList.Serialization.cs:17]: Should the suffix be PageList or ListResult? There are other places where ListResult is used as suffix for similar concept eg. LoadTestingQuotaListResult. The suffix should be consistent?

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ irregular-plural-words:
   quota: quota
 
 rename-mapping:
-  LoadTestResource: LoadTesting
+  Resource: LoadTestingBaseResource
+  LoadTestResource: LoadTestingResource
   QuotaResource: LoadTestingQuota
   CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailabilityResponse
```
Comment (prativen) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:26]: CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailability ?

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md [hunk 3]
```diff
@@ -20,7 +20,8 @@ irregular-plural-words:
...
   EncryptionProperties: CustomerManagedKeyEncryptionProperties
```
Comment (prativen) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: QuotaBucketRequest: LoadTestingQuotaBucket?

Comment (prativen) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: OutboundEnvironmentEndpointCollection: OutboundEnvironmentEndpointPageList ?

Comment (prativen) [inline sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: QuotaResourceList: LoadTestingQuotaPageList?

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/Generated/Models/LoadTestResourcePageList.Serialization.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/Generated/Models/LoadTestResourcePageList.Serialization.cs:17]: This is internal function that don't need to change the name

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:26]: This is a common naming pattern for all CheckAvailability method and related models

File: sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md (hunk 3 above)
Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: cThis is a common naming pattern for a model which is used as a method parameter, end with `Content` suffix.

Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: Here the `Collection` is a reserved suffix for ResourceCollection, so here rename it, and the naming pattern is change the suffix from `Collection` to `ListResult`.

Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: Similar as above, the suffix is `ListResult` as it used as a result / response of a method.

Comment (prativen) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: @ArthurMa1978 in that case, should LoadTestResourcePageList be renamed to LoadTestResourceListResult

Comment (ArthurMa1978) [reply sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md:27]: The `LoadTestResourcePageList` is internal class that we don't need to rename, the rename function only works for all public class / interface / method.

**Pull request merged**